chargeOclock/Source/Models/Client.swift

```diff
@@ -32,6 +32,12 @@ class Client {
 			completion(error)
 		}
 	}
+
+	static func addClient(name: String, completion: @escaping (Error?, Data?) -> Void) {
```

Comment:
add a isDirty boolean member to the Client model, change existing properties (just name in this case) to set that flag on change, then craete a method named `save`, that way we have a way to both update clients and create new ones.